src/components/pagination/__tests__/CdrPagination.spec.js

```diff
@@ -281,20 +281,40 @@ describe('CdrPagination', () => {
     expect(wrapper.emitted().navigate[5][0]).toBe(2);
     expect(wrapper.emitted().navigate[5][1]).toBe('?page=2');
     expect(wrapper.emitted().navigate[5][2] instanceof Event).toBeTruthy();
+    
     // Do nothing when clicking current page link
+    const originalOffsetHeight = Object.getOwnPropertyDescriptor(HTMLElement.prototype,'offsetHeight');
```

Comment:
This felt a bit clunky, but this is what stack overflow recommended as a way to provide offsetHeight and width for the pagination links. It appears in that in this environment they are either zero or undefined.